samples/Catalog/Catalog.ios.js

```diff
@@ -593,6 +593,7 @@ class ProgrammaticAnnotations extends Component {
                   },
                   opacity: 1,
                   pageIndex: 0,
+                  name: "A167811E-6D10-4546-A147-B7AD775FE8AC",
```

Comment:
Do we need to specify this? Shouldn't this be set automatically when creating the annotation?

Reply:
In the case of this example, yes. Because we need the annotation name to ensure the uniqueness of the annotation. See https://pspdfkit.com/api/ios/Classes/PSPDFAnnotation.html#/c:objc(cs)PSPDFAnnotation(py)name and https://github.com/PSPDFKit/PSPDFKit/issues/17635